.circleci/config.yml

```diff
@@ -187,31 +206,27 @@ jobs:
       - restore_cache:
           name: Restore Yarn cache
           keys:
-            - yarn-packages-v1-{{ checksum "yarn.lock" }}
+            - yarn-packages-v2-{{ checksum "yarn.lock" }}
       - run:
           name: Download dependencies
           command: yarn install --frozen-lockfile --ignore-optional --non-interactive
+      # - run: yarn test
+      # - run: ./node_modules/.bin/codecov -p ..
+      - run: yarn build
+      - run: yarn lint
+      - run: git diff --exit-code
       - save_cache:
           name: Save Yarn cache
-          key: yarn-packages-v1-{{ checksum "yarn.lock" }}
+          key: yarn-packages-v2-{{ checksum "yarn.lock" }}
           paths: [~/.cache/yarn, node_modules]
-      # - run: yarn test
-      # - run: ./node_modules/.bin/codecov -p ..
-      - run:
-          name: Build
-          command: yarn build
-      - run:
-          name: Lint
-          command: yarn lint
-      - run:
-          name: Check nothing needs committing
-          command: git diff --exit-code
+          when: always
 workflows:
   version: 2
-  workflow:
+  argo-workflow:
```

Comment:
change the name to make it easier to identify in a UI that uses the term "workflow" a lot